pulsar-client-cpp/lib/LogUtils.h

```diff
@@ -27,7 +27,11 @@
 
 namespace pulsar {
 
+#if defined _WIN32 || defined _WIN64
```

Comment:
how can we test if this works for windows c++ compiler? can we have a docker based test or vagrant based test?